cmd/client/wallet/main.go

```diff
@@ -608,6 +610,19 @@ func getBlsPublic() {
 			os.Exit(101)
 		}
 		fmt.Printf("Your bls public key is: %s\n", privateKey.GetPublicKey().SerializeToHexStr())
+	} else if *blsFile2 != "" {
+		password := utils.AskForPassphrase("Passphrase: ")
+		password2 := utils.AskForPassphrase("Passphrase again: ")
+		if password != password2 {
+			fmt.Printf("Passphrase doesn't match. Please try again!\n")
+			os.Exit(100)
+		}
+		privateKey, err := blsgen.LoadBlsKeyWithPassPhrase(*blsFile2, password)
+		if err != nil {
+			fmt.Printf("error when loading bls key, err :%v\n", err)
+			os.Exit(100)
+		}
+		fmt.Printf("Your bls public key is: %s\n", privateKey.GetPublicKey().SerializeToHexStr())
```

Comment:
After speaking with John, it seems he wants the user to check whether this public key matches what the user has with them.  Can you just change the printf statement to "Your generate bls public key is: ... It should match what you have".

Reply:
discussed offline